workers/unity/Assets/Playground/Scripts/UI/InitUIBehaviour.cs

```diff
@@ -12,21 +12,25 @@ public class InitUIBehaviour : MonoBehaviour
         [Require] private ScoreReader score;
         [Require] private LauncherReader launcher;
 
+        [SerializeField] private Object uiPrefab;
```

Comment:
This should be a `GameObject` for a prefab.

Reply:
is there a reason for this besides convention?

Reply:
well I guess it removes the need to cast after instantiating it

Reply:
It acts as a restriction in the inspector. With `GameObject` only gameobjects and prefabs can be dragged in. With `Object` any Unity object will do, textures, etc.

Reply:
Its also the more specific type